src/index.html

```diff
@@ -13,6 +13,76 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="stars--0">
```

Comment:
Class names should represent the meaning of the content (not the styles or tag names). In your case, 'stars--0' doesn't seem to represent the content. Try to use more meaningful class names.